packages/react/src/components/Layout/Grid-story.js

```diff
@@ -0,0 +1,256 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import './story.scss';
+
+import { storiesOf } from '@storybook/react';
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import {
+  unstable_Grid as Grid,
+  unstable_Row as Row,
+  unstable_Column as Column,
+} from '../Layout';
+
+function DemoFullPage({ children }) {
+  const [portalNode, setPortalNode] = useState(null);
+
+  useEffect(() => {
+    const node = document.createElement('div');
+    document.body.appendChild(node);
+
+    setPortalNode(node);
+
+    return () => {
+      document.body.removeChild(node);
+    };
+  }, []);
+
+  return (
+    portalNode &&
+    ReactDOM.createPortal(<div className="example">{children}</div>, portalNode)
+  );
+}
+
+function DemoContent({ children }) {
+  return (
+    <div className="outside">
+      <div className="inside">{children}</div>
+    </div>
+  );
+}
+
+storiesOf('Layout/Grid', module)
+  .addDecorator(story => <DemoFullPage>{story()}</DemoFullPage>)
+  .add('auto-columns', () => (
+    <Grid>
+      <Row>
+        <Column>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+        <Column>
+          <DemoContent>1/4</DemoContent>
+        </Column>
+      </Row>
+    </Grid>
+  ))
+  .add('responsive grid', () => (
+    <Grid>
+      <Row>
+        <Column span={[1, 4, 8]}>
+          <DemoContent>1/4</DemoContent>
```

Comment:
Looking at some of these in the preview, it is a little confusing:

![Screen Shot 2019-09-03 at 2 17 07 PM](https://user-images.githubusercontent.com/9057921/64202021-b5d9c280-ce55-11e9-96d2-0556ea158d1b.png)

When I first saw how large the first column in the preview was, even though it said `1/4`, I wasn't sure what was meant. At first I thought something was wrong until I looked at the values in the `span` prop.

Is `1/4` meant to illustrate "1 of 4 columns"? 
Would possible to put more information about the `span` inside the `DemoContent` instead?